README.md

```diff
@@ -1,11 +1,22 @@
-# Spark Monorepo
+<img src="https://studio-assets.supernova.io/design-systems/27408/b8ad8938-c463-41aa-ad30-c659b149b853.png"/>
 
-_Visit the [V4 migration guide](docs/migration-guides/v4/readme.md) for details on the new v4 release and migration details._
+# Spark Design System
 
-This project contains several packages that make up Genesys' design system, Spark.
+[![Major Version](https://img.shields.io/badge/V4-Flare-orange)](docs/migration-guides/v4/readme.md)
+[![NPM latest](https://img.shields.io/npm/v/genesys-spark/latest.svg)](docs/migration-guides/v4/readme.md)
+[![License](https://img.shields.io/badge/license-MIT-blue)](https://github.com/MyPureCloud/genesys-spark/blob/main/LICENSE)
+[![Documentation](https://img.shields.io/badge/documentation-8A2BE2)](https://spark.genesys.com?utm_source=github&utm_medium=readme_shield)
+[![Figma library](https://img.shields.io/badge/spark_core_components-darkslateblue?logo=figma&logoColor=white)](https://www.figma.com/file/JKbHmcf4nUF6C7Pj8M6MpY)
+
+> ⚠ Visit the [V4 migration guide](docs/migration-guides/v4/readme.md) for details on the new v4 release and migration details.
+
+Spark is Genesys' Design System it contains all our styles and components.  
+**Visit [https://spark.genesys.com/](https://spark.genesys.com?utm_source=github&utm_medium=readme_p) for our full documentation in all things Spark.**
```

Comment:
Given that this is the development repo for the webcomponets, I don't think the first thing we should do is direct folks away from the project to a documentation site for "all things spark" that doesn't have documentation for

- how to contribute to the project
- development best practices
- how to use the components from this project

IMO, it would be better to break this down into several sections, things like:

"Keep reading for an overview of the Spark packages and links on how to get started installing them"
"If you want to view examples of component usage and try the components out, go to our live editor sandbox"
"For details on component design and usage guidelines, go to spark.genesys.com"
"If you want to contribute to the codebase, check out the contributing guide"

It would be nice eventually to have all of those in one place, but until we do, let's not hide them from people.

Reply:
Thanks for the feedback Matt! how about something like:

- Keep reading for an overview of the Spark packages and links on how to get started installing them.
- If you want to view examples of component usage and try the components out, go to our live editor sandbox
- For details on component documentation and guidelines, go to spark.genesys.com 
- If you want to contribute to the codebase, check out the contributing guide!


Now that the code docs and sandbox can be also found in [our site](https://spark.genesys.com/latest/components/search-and-filter/search/standard/code-ShPeM2pL).

Reply:
@VirtualClover On a side note, I think the code pages on Suprenova should also link to the relevant "common-ui-docs" page as its easier for devs to work with.

Reply:
@MattCheely @conor-darcy 
Changes added :)

Reply:
Hi @MattCheely 👋🏼 , just checking in, have the changes that I made make sense?